.rhcicd/clowdapp-backend.yaml

```diff
@@ -183,25 +183,64 @@ objects:
       secretName: ${FLOORIST_BUCKET_SECRET_NAME}
     suspend: ${{FLOORIST_SUSPEND}}
     queries:
-    - prefix: insights/notifications/splunk
+    # Count the number of created endpoints per endpoint type. Raw total, which
+    # includes the disabled ones or the ones not assigned in a behavior group.
+    - prefix: insights/notifications/endpoint_types
       query: >-
         SELECT
-          DISTINCT "account_id"
-          FROM "endpoints"
-          WHERE "endpoint_type_v2"='CAMEL' AND "endpoint_sub_type"='splunk';
-    - prefix: insights/notifications/accounts
+          endpoint_type_v2 AS "Endpoint type",
+          count(endpoint_type_v2) AS "Count"
+        FROM
+          endpoints
+        GROUP BY
+        endpoint_type_v2;
```

Comment:
I think we should include the endpoints subtype somehow in the queries. Each subtype of the `CAMEL` type should have its own record in the query result. And because the whole CAMEL / not CAMEL logic won't make any sense for someone outside of our team, it'd be better if we simply exported one "type" column showing directly the kind of endpoint.

Reply:
Here's what I meant:

| Endpoint type | Count |
| - | - |
| email | 3 |
| slack | 12 |
| webhook | 7 |
| splunk | 42 |

Reply:
Untested:
```sql
        SELECT
          CASE
            WHEN endpoint_type_v2 = 'CAMEL' THEN endpoint_sub_type
            ELSE endpoint_type_v2
          END AS "Endpoint type",
          count(endpoint_type_v2) AS "Count"
        FROM
          endpoints
        GROUP BY
        endpoint_type_v2;
```

Reply:
Updated!